cpp/include/cugraph/algorithms.hpp

```diff
@@ -1536,6 +1537,32 @@ uniform_nbr_sample(raft::handle_t const& handle,
                    std::vector<int> const& h_fan_out,
                    bool with_replacement = true);
 
+/**
+ * @brief Multi-GPU Uniform Neighborhood Sampling.
```

Comment:
Is this really a Multi-GPU only thing or for both SG & MG

Reply:
Both.  Updated the comment.